packages/mask/shared-ui/locales/en-US.json

```diff
@@ -1111,5 +1111,85 @@
     "plugin_transak_name": "Fiat On-Ramp",
     "plugin_transak_description": "Fiat On-Ramp Aggregator on Twitter. Buy crypto in 60+ countries with Transak support.",
     "plugin_valuables_name": "Valuables",
-    "plugin_valuables_description": "Buy & sell tweets autographed by their original creators."
+    "plugin_valuables_description": "Buy & sell tweets autographed by their original creators.",
+    "plugin_referral": "Referral Farms",
+    "plugin_referral_powered_by": "Powered by",
+    "plugin_referral_create_by": "Created by",
+    "plugin_referral_tab_new": "New",
+    "plugin_referral_tab_created": "Created",
+    "plugin_referral_tab_my_rewards": "My Rewards",
+    "plugin_referral_referral_farming": "Referral Farming",
+    "plugin_referral_referral_farms_short_desc": " distributes yield farming alike returns for successful referrals",
+    "plugin_referral_how_it_works": "How it Works",
+    "plugin_referral_select_a_token_desc": "Select a token to be promoted and define daily rewards to participants, as well as total referral farm rewards.",
+    "plugin_referral_create_referral_farm_desc": "Create a farm to incentivise promotion of a token",
+    "plugin_referral_daily_farm_reward": "Daily Farm Reward",
+    "plugin_referral_total_farm_rewards": "Total Farm Rewards",
+    "plugin_referral_daily_reward": "Daily Reward",
+    "plugin_referral_total_rewards": "Total Rewards",
```

Comment:
Manage in its own locales folders:

You can follow `packages/mask/src/plugins/MaskBox` for example.

1. Move to `packages/mask/src/plugins/Referral/locales`. (Don't forget to remove prefix `plugin_referral`)
2. Append i18n config in `.i18n-codegen.json`, and execute `npx gulp i18n-codegen` for ts typing and runtime code. Make sure the namespace is same to the id of the plugin.
3. Register i18n in `Referral/base.ts` by add `i18n: languages`.

Reply:
Thanks, done